scopus-import/src/main/java/no/sikt/nva/scopus/conversion/PiaConnection.java

```diff
@@ -24,33 +23,36 @@
 import no.unit.nva.commons.json.JsonUtils;
 import nva.commons.core.Environment;
 import nva.commons.core.JacocoGenerated;
+import nva.commons.core.paths.UriWrapper;
 import nva.commons.secrets.SecretsReader;
-import org.apache.http.client.utils.URIBuilder;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class PiaConnection {
-
-    public static final String CRISTIN_PERSON_PATH = "/cristin/person/";
+    public static final String CRISTIN_PATH = "cristin";
+    public static final String PERSON_PATH = "person";
     public static final int FALSE_IN_PIA_INTEGER = 0;
     public static final String PIA_REST_API_ENV_KEY = "PIA_REST_API";
     public static final String API_HOST = "API_HOST";
     public static final String PIA_USERNAME_KEY = "PIA_USERNAME_KEY";
     public static final String PIA_PASSWORD_KEY = "PIA_PASSWORD_KEY";
     public static final String PIA_SECRETS_NAME_ENV_KEY = "PIA_SECRETS_NAME";
-    public static final String PIA_AUTHORS_PATH = "/sentralimport/authors";
-    public static final String PIA_ORGS_PATH = "/sentralimport/orgs/matches";
+    public static final String PIA_PATH = "sentralimport";
+    public static final String PIA_AUTHORS_PATH = "authors";
+    public static final String PIA_ORGS_PATH = "orgs";
+    public static final String PIA_MATCHES_PATH = "matches";
     public static final String PIA_AUTHOR_ID_QUERY_PARAM = "author_id";
     public static final String PIA_AFFILIATION_ID_QUERY_PARAM = "affiliation_id";
     public static final String SCOPUS = "SCOPUS:";
-    public static final String HTTPS_SCHEME = "https";
```

Comment:
The overhead of this line cannot be underestimated. It caused trouble pretty much all the way